setup.py

```diff
@@ -53,8 +54,7 @@
 
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
```

Comment:
Python version should be consistent everywhere. In environment.yml it is also 3.6, in travis.yml it is 3.7. We will go with 3.7 as it also introduces dataclasses (https://docs.python.org/3/library/dataclasses.html) which we might want to use.